analyses/snv-callers/README.md

```diff
@@ -0,0 +1,211 @@
+# SNV caller comparison analysis
+
+This analysis evaluates [MAF files](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)
```

Comment:
I would expect this document to follow our one sentence per line convention. It seems like some of the lines follow it, some of them appear to follow word wrap.

Reply:
Ah yes. I wasn't sure what our rule was for this repository. I'll change it to one sentence per line.